packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -1,9 +1,7 @@
+/* eslint-disable no-unused-vars */
```

Comment:
This eslint-disable should be removed.

Reply:
[Removed](https://github.com/cerner/terra-framework/pull/1563/commits/bafb9129abf5f0b7aa459d3567ccb02523fbdd58)